src/App.tsx

```diff
@@ -1,7 +1,9 @@
 import { DeskproAppProvider } from "@deskpro/app-sdk";
+import { StoreProvider } from "./context/StoreProvider";
 import { Main } from "./pages/Main";
 import "./App.css";
 
+import "iframe-resizer/js/iframeResizer.contentWindow.js";
```

Comment:
You don't need to do this as it's included here: https://github.com/DeskproApps/teamviewer/blob/master/index.html#L7

Reply:
done